lib/gitlab/ci/templates/pipeline/AndroidTemplate.yml

```diff
@@ -258,3 +264,61 @@ workflow:
     - if: $CI_COMMIT_TAG # if this is a release
     - if: $CI_COMMIT_BRANCH == $CI_DEFAULT_BRANCH # if this is a push to develop
     - if: $CI_PIPELINE_SOURCE == "web" # if triggered from the Gitlab UI
+
+combineCoverageReports:
+  stage: report_processing
+  needs: [testDebug, instrumentationTests]
+  script:
+    - echo "Combining test reports using JaCoCo"
+    - ./gradlew $BUILD_TARGET
+    - cat $JACOCO_HTML_LOCATION | grep -o 'Total[^%]*%'
+  artifacts:
+    when: always
+    paths:
+      - $JACOCO_HTML_LOCATION
+      - $JACOCO_XML_LOCATION
```

Comment:
Remove both of these paths and just put `- $JACOCO_OUTPUT_DIR`. Doing this  will place all the Jacoco html files in the Gitlab job artifact, which allows a developer to navigate and visualize way more information about the test coverage then just the Jacococ `index.html` provides.